tools/wpt/browser.py

```diff
@@ -416,7 +420,33 @@ class FirefoxAndroid(Browser):
     requirements = "requirements_firefox.txt"
 
     def install(self, dest=None, channel=None):
-        raise NotImplementedError
+        if dest is None:
+            dest = os.pwd
+
+        TC_QUEUE_BASE = "https://queue.taskcluster.net/v1/"
+        TC_INDEX_BASE = "https://index.taskcluster.net/v1/"
```

Comment:
FYI this will be the first dependency on Taskcluster domain in this repo.

Reply:
Yeah, it's not ideal. If I find somewhere else that this is mirrored to I'll use that instead.